### PR TITLE
#301 + #302 - added involved and vehicles data reader to the process.py - now need to cre…

### PR DESCRIPTION
**Pull request by aviklai**
…ate the dictionaries for the data and integrate the data in the markers + small layout fix in the admin area.

### CODE REVIEW

**Comment by omerxx** [comment]
@aviklai looks great!
I think you'd wanna merge this into 'dev' branch though..

**Comment by aviklai** [comment]
@omerxx I wanted to but I was already on the master. I need to rebase onto the dev but there are too many other changes that are already there. Maybe I will try to do it tomorrow. Anyway, It's not finished because we need to add the dictionaries and add it to the marker view.

File: models.py [hunk 1]
```diff
@@ -315,6 +315,115 @@ def bounding_box_query(ne_lat, ne_lng, sw_lat, sw_lng):
...
+    home_municipal_status = Column(Integer)
+    home_residence_type = Column(Integer)
+    file_type = Column(Integer)
+
+    def serialize(self):
+        return {
+            "id": self.id,
+            "accident_id": self.accident_id,
+            "accident_year": self.accident_year,
+            "month": self.month,
```
Comment (omerxx) [inline models.py]: @aviklai I think we can spare the accident year and month here.
We already have these in the accidents table

File: models.py [hunk 2]
```diff
@@ -315,6 +315,115 @@ def bounding_box_query(ne_lat, ne_lng, sw_lat, sw_lng):
...
+            "injury_severity": self.injury_severity,
+            "injured_type": self.injured_type,
+            "injured_position": self.injured_position,
+            "population_type": self.population_type,
+            "home_district": self.home_district,
+            "home_nafa": self.home_nafa,
+            "home_area": self.home_area,
+            "home_municipal_status": self.home_municipal_status,
+            "home_residence_type": self.home_residence_type,
+            "file_type": self.file_type
```
Comment (omerxx) [inline models.py]: `file_type` don'y need a column, it's just concatenated to the original accident id.
We must concatenate them in order to combine both `Marker` and `Involved` tables correctly.

**Comment by aviklai** [comment]
I know. I noticed that as well. About the file type, I already the concatenation and I can remove it from the vehicle table as well.

File: models.py [hunk 3]
```diff
@@ -315,6 +315,115 @@ def bounding_box_query(ne_lat, ne_lng, sw_lat, sw_lng):
...
+    home_city = Column(Integer)
+    injury_severity = Column(Integer)
+    injured_type = Column(Integer)
+    injured_position = Column(Integer)
+    population_type = Column(Integer)
+    home_district = Column(Integer)
+    home_nafa = Column(Integer)
+    home_area = Column(Integer)
+    home_municipal_status = Column(Integer)
+    home_residence_type = Column(Integer)
```
Comment (omerxx) [inline models.py]: These 5 last columns and `home_city` can all be combined into one string like `home_inv` or something?

**Pull request closed without merging**